helper/config/duration.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"fmt"
+	"time"
+)
+
+// DurationString is a string that represents a time duration.
+//
+// A DurationString is validated using time.ParseDuration.
+//
+// An empty string ("") is a valid (0) DurationString. A time.Sleep(0) returns
+// immediately.
+type DurationString string
+
+// Duration returns the parsed duration.
+// Duration panics if d is invalid.
+func (d DurationString) Duration() time.Duration {
+	if d == "" {
+		return 0
+	}
+	du, err := time.ParseDuration(string(d))
+	if err != nil {
+		s := fmt.Sprintf("DurationString: Could not parse '%s' : %v", d, err)
+		panic(s)
```

Comment:
Ah wait; this is a bad solution because : 
* if a users passes a wrong string and a builder doesn't validate the duration string; packer will crash ( currently packer errors correctly at validation )
* having `Duration()` return an error would be annoying since we want this to be validated at parsing/prepare time.
* when passing a string; time.Duration validates itself.

I think using the original solution (just a time.Duration) is simply the best solution and trying to prevent users from passing and integer here would be like opening a can of worms.

I will revert this new type addition.